tests/unit/test_resolver.py

```diff
@@ -93,3 +97,74 @@ def fetch(req: Request) -> RequestOutput:
         assert resolver.resolve_packages(request) == COMBINED_OUTPUT
 
     assert calls_by_pkgtype == ["gomod", "npm", "pip"]
+
```

Comment:
I'll need a bit of guidance here as still don't have enough insights into the guts of the project - shouldn't we actually test the CLI flag as part of CLI tests and not the resolver? Or maybe at both places?

Reply:
It should be both, IMO. It is already being tested in the [cli](https://github.com/containerbuildsystem/cachi2/blob/76a218fbdc867f27ff7858ff07d3d18662560f6c/tests/unit/test_cli.py#L537-L538). The test is very simple because we're basically validating that the CLI is able to receive the flag. The bulk of the feature is being tested in this resolver test here.

Reply:
> It is already being tested in the [cli](https://github.com/containerbuildsystem/cachi2/blob/76a218fbdc867f27ff7858ff07d3d18662560f6c/tests/unit/test_cli.py#L537-L538).

Ah, so I'm just blind.